pkg/config/imports_test.go

```
@@ -0,0 +1,165 @@
+		assert.Contains(t, resolved, nestedLocalConfigPath, "should resolve nested local imports")
+	})
+
+	t.Run("Test mixed imports with depth limit", func(t *testing.T) {
+		tempDir, err := os.MkdirTemp("", "config-test")
+		assert.NoError(t, err)
+		defer os.RemoveAll(tempDir)
+		importPaths := []string{
+			"local.yaml",
+			server.URL + "/config.yaml",
+		}
+		resolved, err := cl.processImports(importPaths, tempDir, 11, 10)
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)